### PR TITLE
India NHM Data: Uploaded Child Health, Birth Control & General Stats datasets

### PR DESCRIPTION
**Pull request by SenthamizhanV**
@pradh Created a new PR for other three NHM datasets. Kindly take a look at it.

### CODE REVIEW

File: scripts/india_nhm/base/data_cleaner.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pandas as pd
+
+INDIA_ISO_CODES = {
```
Comment (ajaits) [inline scripts/india_nhm/base/data_cleaner.py:18]: Can these codes be merged into 
scripts/india/geo/states.py and then use get_state_name_to_iso_code_mapping(state_name)?

File: scripts/india_nhm/base/data_cleaner.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+        self.raw_df = None
+
+        # Ignoring the first 3 elements (State, isoCode, Date) in the dictionary map
+        self.cols_to_extract = list(self.cols_dict.keys())[3:]
+
+    def generate_csv(self):
+        """
+        Class method to preprocess the data file for each available year, extract t
+        he columns and map the columns to schema.
```
Comment (ajaits) [inline scripts/india_nhm/base/data_cleaner.py:133]: can like line split t\nhe happen at a word boundary? Is there a pylint tool that could automatically wrap such line?

File: scripts/india_nhm/base/data_cleaner.py [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+value: C:{dataset_name}->indianNHM/{statvar}
+"""
+
+MCF_NODES = """Node: dcid:indianNHM/{statvar}
+description: "{description}"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcs:indianNHM/{statvar}
+statType: dcs:measuredValue
+
```
Comment (ajaits) [inline scripts/india_nhm/base/data_cleaner.py:101]: for my understanding: this is a schemaless import with the properties encoded in the statvar name?

File: scripts/india_nhm/base/data_cleaner.py (hunk 1 above)
Comment (SenthamizhanV) [reply scripts/india_nhm/base/data_cleaner.py:18]: I think Thej is working on creating a geos folder. Not sure about that. If he is not, I can create it and implement this.

File: scripts/india_nhm/base/data_cleaner.py (hunk 2 above)
Comment (SenthamizhanV) [reply scripts/india_nhm/base/data_cleaner.py:133]: It is the 'yapf' package for format checks. We mention the code style to be 'google' style which allows only 80 characters per line. It automatically does this wrap for comments. I will look into the abnormal comments like this. Thanks for identifying this.

File: scripts/india_nhm/base/data_cleaner.py (hunk 3 above)
Comment (SenthamizhanV) [reply scripts/india_nhm/base/data_cleaner.py:101]: Yes. You are right.

**Pull request closed without merging**